onnx/defs/tensor/defs.cc

```diff
@@ -1801,4 +1801,129 @@ ONNX_OPERATOR_SET_SCHEMA(
 
           propagateShapeFromInputToOutput(ctx, 0, 0);
         }));
+
+static const char* Unique_ver11_doc = R"DOC(
+Find the unique elements of a tensor. When an optional attribute 'axis' is provided, unique subtensors along the 'axis' are returned. 
+Otherwise the input tensor is flattened and unique values of the flattened tensor are returned. 
+
+This operator returns the unique elements of the input tensor and three optional outputs. 
+The first output tensor 'Y' contains all unique values or subtensors of the input. 
+The second optional output tensor 'indices': indices[j] contains the index of the first occurrence of Y[j] in input X. 
+The third optional output tensor 'inverse_indices': inverse_indices[j] contains the index of the occurrence of X[j] in the output Y. 
+The fourth optional output tensor 'counts' contains the count of each element of 'Y' in the input. 
+
+Outputs are either sorted in ascending order or optionally in the order they occur in the input. 
```

Comment:
"in the order they occur in the input" => "in the order of the first occurrence of the values in the input" ?